DFGM/dfgm_test_receiver.py

```diff
@@ -0,0 +1,88 @@
+"""This python program represents a simulated part of the OBC meant to handle incoming data
+from the DFGM payload on ExAlta3.
+
+The program is meant to be used along with dfgm_subsystem.py and mainly serves as a way to
+see how data can be received and read from the subsystem.
+
+The program also utilizes the local host IP '127.0.0.1'.
+
+Usage: dfgm_test_receiver.py
+
+Copyright 2024 [Daniel Sacro]. Licensed under the Apache License, Version 2.0
+"""
+
+import socket
+
+DEFAULT_HOST = '127.0.0.1'
+DEFAULT_PORT = 1802
+
+with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+    s.connect((DEFAULT_HOST, DEFAULT_PORT))
+    while True:
+        data = s.recv(1248)
```

Comment:
This needs to be set to 1252 otherwise the program crashes with an 'index out of range' error. Not sure why but it should be explored before merge.

Reply:
Also I think its best practice to avoid 'magic numbers' and make something like the buffer size a const.